client/components/document-share.vue

```diff
@@ -0,0 +1,459 @@
+                <translate>document-visibility</translate>
+                <small><translate>document-visibility-hint</translate></small>
+              </v-subheader>
+
+              <v-list-tile
+                v-for="level in visibilityLevels"
+                :key="level.value"
+                ripple
+                @click="visibilityLevel = level.value"
+              >
+                <v-list-tile-action>
+                  <v-radio
+                    :key="level.value"
+                    :value="level.value"
+                  />
+                </v-list-tile-action>
+                <v-list-tile-content
+                  @click="visibilityLevel = level.value"
+                >
+                  <v-list-tile-title>{{level.label}}</v-list-tile-title>
+                  <v-list-tile-sub-title>{{level.hint}}</v-list-tile-sub-title>
+                </v-list-tile-content>
+              </v-list-tile>
+            </v-list>
+          </v-radio-group>
+        </v-card-text>
+
+        <v-divider />
+
+        <v-card-text>
+          <v-radio-group
+            v-model="defaultPermissions"
+            class="document-share__radio-group"
+            hide-details
+          >
+            <v-list two-line>
+              <v-subheader class="document-share__subheader--with-hint">
+                <translate>document-default-permissions</translate>
+                <small><translate>document-default-permissions-hint</translate></small>
+              </v-subheader>
+
+              <v-list-tile
+                v-for="permission in defaultPermissions"
+                :key="permission.value"
+                ripple
+                @click="defaultPermission = permission.value"
+              >
+                <v-list-tile-action>
+                  <v-radio
+                    :key="permission.value"
+                    :value="permission.value"
+                  />
+                </v-list-tile-action>
+                <v-list-tile-content
+                  @click="defaultPermission = permission.value"
+                >
+                  <v-list-tile-title>{{permission.label}}</v-list-tile-title>
+                  <v-list-tile-sub-title>{{permission.hint}}</v-list-tile-sub-title>
+                </v-list-tile-content>
+              </v-list-tile>
+            </v-list>
+          </v-radio-group>
+        </v-card-text>
+
+        <v-divider />
+
+        <v-card-text>
+          <v-list two-line>
+            <v-subheader class="document-share__subheader--with-hint">
+              <translate>document-user-permissions</translate>
+              <small><translate>document-user-permissions-hint</translate></small>
+            </v-subheader>
+
+            <v-list-tile
+              v-for="contributor in contributors"
+              :key="contributor._id"
+              avatar
+            >
+              <v-list-tile-avatar>
+                <img :src="contributor.avatar">
```

Comment:
This is not correct. You should never access `avatar` directly, but use `avatarUrl`. Becaue it might be that `avatar` is just a local filename or something, which is then resolved to an URL.